common/src/main/java/com/github/streamshub/console/config/security/OidcConfig.java

```diff
@@ -22,6 +24,8 @@ public class OidcConfig implements Trustable {
     private String clientId;
     @NotBlank
     private String clientSecret;
+    @NotBlank
+    private List<String> roleClaimPath = List.of("groups"); 
```

Comment:
We can actually drop the default of `"groups"`, it's already assumed by the Quarkus extension as the claim path for the roles.

```suggestion
    private List<@NotBlank String> roleClaimPath; 
```

Reply:
Done